src/polyswarmclient/__init__.py

```diff
@@ -330,7 +333,9 @@ def sign_transactions(self, transactions):
         if not is_valid_ipfs_uri(ipfs_uri):
             raise ValueError('Invalid IPFS URI')
 
-        uri = urljoin(self.polyswarmd_uri, '/artifacts/{0}/{1}'.format(ipfs_uri, index))
+        uri = self.polyswarmd_uri + '/artifacts/{0}/{1}'.format(ipfs_uri, index)
```

Comment:
Can we convert this to f string
```suggestion
        uri = f'{self.polyswarmd_uri}/artifacts/{ipfs_uri}/{index}'
```
https://github.com/polyswarm/polyswarm-client/blob/fe4deb4a52862c6f1ab91d3784249a9e8cb52911/src/worker/__init__.py#L103